### PR TITLE
Add info text to person pages #409

### PR DESCRIPTION
**Pull request by Phu2**

### CODE REVIEW

File: app/models/AuthorityResource.java [hunk 1]
```diff
@@ -187,6 +187,33 @@ public String gndRelationEdges() {
 		return Json.toJson(result).toString();
 	}
 
+	public Integer getBirthYear() {
+		String[] parts = fieldValues("dateOfBirth-dateOfDeath", json)
+			.collect(Collectors.joining())
+			.split("–");
+		return Integer.parseInt(parts[0]);
```
Comment (fsteeg) [inline app/models/AuthorityResource.java:194]: Instead of first getting the composed form `dateOfBirth-dateOfDeath` and splitting that, we can get the `dateOfBirth` directly using `fieldValues("dateOfBirth", json)`. We also need to make sure to handle missing birth dates here (see e.g. https://test.lobid.org/gnd/1359893474). Finally, like you did in `getDeathYear()`, we should get only the year, but we can use the existing `year()` function for that. So e.g.:

```suggestion
		return fieldValues("dateOfBirth", json).findFirst()
				.map(AuthorityResource::year)
				.map(Integer::parseInt)
				.orElse(null);
```

File: app/models/AuthorityResource.java [hunk 2]
```diff
@@ -187,6 +187,33 @@ public String gndRelationEdges() {
...
+			.collect(Collectors.joining())
+			.split("–");
+		if (parts.length > 1 && parts[1].length() == 4) {
+			try {
+				return Integer.parseInt(parts[1]);
+			} catch (NumberFormatException e) {
+				return null;
+			}
+		}
+		return null;
```
Comment (fsteeg) [inline app/models/AuthorityResource.java:208]: Like above for `getBirthYear()`:

```suggestion
		return fieldValues("dateOfDeath", json).findFirst()
				.map(AuthorityResource::year)
				.map(Integer::parseInt)
				.orElse(null);
```

At this point, we could extract these two to a single method, passing the field name. There, we could also handle the NumberFormatException. To test these edge cases (missings fields, NumberFormatException, etc.), we should probably add some unit tests in `test/models/ModelTest.java`.

File: app/models/AuthorityResource.java [hunk 3]
```diff
@@ -187,6 +187,33 @@ public String gndRelationEdges() {
...
+	}
+
+	public String getFirstAndLastName() {
+		String[] lastAndFirstName = preferredName.split(", ");
+		String firstAndLastName = lastAndFirstName[1] + " " + lastAndFirstName[0];
```
Comment (fsteeg) [inline app/models/AuthorityResource.java:213]: Here, I forgot to handle the case where the preferredName does not contain a comma in the implementation I pointed you to recently. See e.g. https://test.lobid.org/gnd/1208647903. I later fixed it like this:

```suggestion
		String firstAndLastName = lastAndFirstName.length == 2
				? lastAndFirstName[1] + " " + lastAndFirstName[0]
				: preferredName;
```

See https://github.com/hbz/rppd/pull/4/files for the original fix, which contains an additional test that we should add here too. We could also add corresponding tests for the missing dates and possible cases triggering the NumberFormatException (in addition or instead of the unit tests mentioned above).

File: app/views/details.scala.html [hunk 4]
```diff
@@ -245,6 +245,41 @@ <h1>
             </small>
         </h1>
     </div>
+    @if(resource.getType.contains("DifferentiatedPerson") && resource.getBirthYear > 1940 && resource.getDeathYear == null){
+    <div id="meta-person" class="panel panel-info">
```
Comment (fsteeg) [inline app/views/details.scala.html:249]: I think the `panel-info` color doesn't look right with our colors. To use our brand color, we can use `panel-primary`, but that is too dark and prominent in my opinion. I think we should use `panel-default`:

```suggestion
    <div id="meta-person" class="panel panel-default">
```

File: app/models/AuthorityResource.java (hunk 3 above)
Comment (Phu2) [reply app/models/AuthorityResource.java]: I added a test which covers the name and missing dates, see https://github.com/hbz/lobid-gnd/pull/429/commits/8526351fae7467cab10196f2ac4d6bb688669860

**Comment by Phu2** [comment]
Regarding the failing tests 

> [error] Failed: Total 309, Failed 2, Errors 0, Passed 307
> [error] Failed tests:
> [error] 	modules.IndexQueryTest
> [error] 	apps.DataWorkflowTests
> [error] (test:test) sbt.TestsFailedException: Tests unsuccessful
> [error] Total time: 224 s, completed Jul 2, 2025 12:59:38 PM

https://github.com/hbz/lobid-gnd/actions/runs/16025637278/job/45212738777?pr=429 
Unfortunately i don't know what is going on here and why they are failing.
